builtin/providers/rundeck/resource_job.go

```diff
@@ -347,22 +415,80 @@ func jobFromResourceData(d *schema.ResourceData) (*rundeck.JobDetail, error) {
 		},
 	}
 
+	// Element: Job>Dispatch
+	dispatchConfigI := d.Get("dispatch").([]interface{})
+	if len(dispatchConfigI) > 1 {
+		return nil, fmt.Errorf("rundeck command may have no more than one dispatch")
+	}
+	if len(dispatchConfigI) > 0 {
+		dispatchMap := dispatchConfigI[0].(map[string]interface{})
+		job.Dispatch.ContinueOnError = dispatchMap["continue_on_error"].(bool)
+	}
+
+	// Element: Job>Schedule
+	scheduleCron := d.Get("schedule_cron").(string)
+	if len(scheduleCron) > 0 {
+		scheduleCronArray := strings.Split(scheduleCron, " ")
+		if len(scheduleCronArray) != 7 {
+			return nil, fmt.Errorf("rundeck schedule_cron format is incorrect")
```

Comment:
As for the ValidateFunc, there are numerous checks done by rundeck. These errors are easy to understand when you do an apply plan. And are easy to find and modify.

In my opinion, i kept the number of logical checks to a minimum and just let rundeck do its job.